packages/cascara/src/ui/Table/Table.js

```diff
@@ -78,109 +77,35 @@ const propTypes = {
 };
 
 /** This is a Table */
-const Table = ({
```

Comment:
All of this logic got moved to TableBase so that we do not try to run any of it during the initial table render if we have an empty or zero length data.